lib/rdoc/parser/ruby.rb

```diff
@@ -145,21 +136,8 @@ class RDoc::Parser::Ruby < RDoc::Parser
 
   parse_files_matching(/\.rbw?$/)
```

Comment:
To make it easier to rollback or compare behaviour between the 2 parsers, can we create a new `parser/prism_ruby.rb` file to host the new parser, and do the same for the test file too?
And to switch we simply remove this line from the current parser and add it to the new one.

Reply:
Updated :+1:

Added parser `Parser::PrismRuby` can be enabled by setting `ENV['RDOC_USE_PRISM_PARSER']`
Added test `test_rdoc_parser_prism_ruby.rb` tests both `Parser::PrismRuby` and `Parser::Ruby` (the above env is not needed for this test)